src/api/world.ts

```diff
@@ -152,6 +155,18 @@ export const createFirestoreWorldAdvancedInput: (
   return { ...picked, radioStations };
 };
 
+export const createFirestoreWorldScheduleInput: (
+  input: WithId<WorldScheduleSettings>
+) => Promise<Partial<World>> = async (input) => {
+  const worldUpdateData: Partial<WithId<World>> = {
```

Comment:
In order to minimize use of `WithId` (it's hard to move it ATM since so many places use it), there is `WorldWithId` type alias. More like it can be added

Reply:
WorldWithId is using `id: WorldId` and the one passed here is `string` the types are a complete mess atm, and we need to refactor them and use the same ones to keep it consistent.